playwright/_impl/_helper.py

```diff
@@ -67,7 +67,7 @@ class ErrorPayload(TypedDict, total=False):
     message: str
     name: str
     stack: str
-    value: Any
+    value: Optional[Any]
```

Comment:
Doesn't total=False already say that all its properties are optional / can be None?

Reply:
Nope it means the keys can be missing but that they are non